easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'fftlib'
+version = '20170628'
+_commit = '44c5fa0'
```

Comment:
underscore-prefixed variables are indeed valid as local variables, but I think using `local_commit` instead is clearer?

Reply:
same for `_options`?

Reply:
Yeah, using `local_*` consistently is better imho